packages/property-grid/src/PropertyGridUiItemsProvider.tsx

```diff
@@ -9,10 +9,11 @@ import {
   StagePanelSection,
   StageUsage,
   UiItemsProvider,
-  WidgetState
+  WidgetState,
 } from "@bentley/ui-abstract";
 import * as React from "react";
-import { FunctionalPropertyGridWidget } from "./components/FunctionalPropertyGridWidget";
+
+import { FunctionalMultiElementPropertyGrid } from "./components/functional";
```

Comment:
do we need to name this "functional"? Theoretically you should be able to refactor a class component into a functional component without breaking changes. Do we need both?

Reply:
Main reason i did this was i didn't want to break exisiting applications and make this a major ver bump with 3.0 around the corner + scope change. The class based components currently have some DR based functionality like featureTracking and debugLogs that i didnt port over. I was originally thinking removing all those and the class based components in the new pkg, but if we want i can make the change now.

Reply:
ok that makes sense. Just not sure we should name components based on their component type. Wonder if another name makes sense that we don't have to change again once the class component is inevitably deprecated...Is it just different because it is missing those few features? If so, maybe something like BasicMultiElementPropertyGrid?

Reply:
Yea basically... (pun inteded 😁), will update component names.

Reply:
Went back and added that one featureTracking call i didnt port over, and then just replaced the class components with the functional one entirely. Added some deprecation tags.